skxray/core/recip.py

```diff
@@ -165,15 +164,15 @@ def process_to_q(setting_angles, detector_size, pixel_size,
                         ccd_cen=(calibrated_center),
                         dist=dist_sample,
                         wavelength=wavelength,
-                        UBinv=np.matrix(ub).I)
-    # **kwargs)
+                        UBinv=np.matrix(ub).I,
+                        n_threads=n_threads)
```

Comment:
put the

```python
if n_threads is None:
   n_threads = 0
```

just above this call.  Understand why you want int input to the function, but 'magic' int values is something we do not have to do in python.